### PR TITLE
Filter courses

### LINKED ISSUE #1921: Explore page should have a course search feature
As the number of courses grows, it becomes less practical to find a particular program you are looking for starting from `/explore`. Being able to search for courses (by matching something in the title or school, or by alternatively searching by instructor/facilitator) would improve the usability of this page considerably.

### PR DESCRIPTION
**Pull request by AmitJoki**
Fixes #1921

### CODE REVIEW

File: app/views/explore/index.html.haml [hunk 1]
```diff
@@ -5,6 +5,11 @@
     %h1= Figaro.env.dashboard_title
 
 .container
+  = form_tag explore_path, class: 'explore-courses', method: :get do
+    = text_field_tag(:search, '', placeholder: "Search the courses")
```
Comment (ragesoss) [inline app/views/explore/index.html.haml]: The placeholder needs to be an i18n messsage, and it needs to switch on `@presenter.course_string_prefix` (as with the table sort controls, for example).

The two versions in the `courses` and `courses_generic` message key sections  should be "Search courses" and "Search programs".

**Comment by ragesoss** [comment]
Nice. With the i18n tweak I noted, this should be good to go.

**Comment by ragesoss** [comment]
Fixed and merged.

**Comment by AmitJoki** [comment]
Didn't know you fixed it, saw the notification and sent a PR fix! 😅

**Pull request closed without merging**